app/src/main/java/com/jerboa/ui/components/home/sidebar/SiteSidebarActivity.kt

```diff
@@ -66,7 +66,14 @@ fun SiteSidebarActivity(
                 is ApiState.Failure -> ApiErrorText(siteRes.msg)
                 ApiState.Loading -> LoadingBar(padding)
                 is ApiState.Success -> {
-                    SiteSidebar(siteView = siteRes.data.site_view, padding = padding)
+                    SiteSidebar(
+                        siteRes = siteRes.data,
+                        showAvatar = siteViewModel.showAvatar(),
+                        onPersonClick = { personId ->
```

Comment:
For these I prefer `appState::toProfile`

Reply:
Good call.